### ISSUE TITLE
not working with FormUrlEncodedContent

### ISSUE THREAD

**Issue by ivanjx** [skwasjer/MockHttp#63]
if i use FormUrlEncodedContent in my service it wont work. it will only work if i use StringContent manually.
service:
```
Dictionary<string, string> data = new()
        {
            { "metadata[id]", userId },
            { "email", email },
            { "name", fullName }
        };
FormUrlEncodedContent content = new FormUrlEncodedContent(data);
```
test:
```
                .FormData(new Dictionary<string, string>()
                {
                    { "metadata[id]", "userid" },
                    { "email", "user@mail.com" },
                    { "name", "John Doe" }
                })
```

the error:
```
MockHttp.HttpMockException : There are 1 unfulfilled expectations:
	RequestUri: 'xxx', Method: POST, FormData: 'metadata%5Bid%5D=userid&email=user%40mail.com&name=John%20Doe'
Seen requests: Method: POST, RequestUri: 'xxx', Version: 1.1, Content: System.Net.Http.FormUrlEncodedContent, Headers:
{
  Authorization: Basic U0VDUkVUOg==
  Content-Type: application/x-www-form-urlencoded
  Content-Length: 59
}
```

**Comment by skwasjer** [comment]
Thanks for reporting, I will try to have a look asap.

**Comment by skwasjer** [comment]
Released under v4.1.0
